mirrulations-client/src/mirrclient/client.py

```diff
@@ -129,6 +129,38 @@ def get_job(self):
 
         return job
 
+    def process_job_results(self, job, job_result):
+        # TODO new function for handling errors
+        # This function is not complete yet, but should replace send_job
+        data = {
+            'job_type': job['job_type'],
+            'job_id': job['job_id'],
+            'results': job_result,
+            'reg_id': job['reg_id'],
+            'agency': job['agency']
+        }
+        print(f'Sending Job {job["job_id"]} to Work Server')
+        if 'error' in job_result:
+            # Handles errors in job_results
+            result = self.redis.hget('jobs_in_progress', job.get('job_id'))
```

Comment:
This code is saying if an error is found, we want to get the url from the `jobs_in_progress` redis hash with `job_id` key and set the `invalid_jobs` redis hash key `job_id` equal to the job_id's corresponding url.

Basically saying `invalid_jobs['job_id'] `= `jobs_in_progress['job_id'] `
Where `invalid_jobs` and `jobs_in_progress` are redis hashes.